CRM/Mailing/BAO/Spool.php

```diff
@@ -90,7 +88,7 @@ public function send($recipient, $headers, $body, $job_id = NULL) {
       }
 
       $job = new CRM_Mailing_BAO_MailingJob();
-      $job->is_test = 0; // if set to 1 it doesn't show in the UI
+      $job->is_test = 0;  // if set to 1 it doesn't show in the UI
```

Comment:
hmm perhaps this line was right & they were wrong.. but jenkins will let us know if it thinks they are not right